snap/snapcraft.yaml

```diff
@@ -138,6 +187,12 @@ parts:
       curl -o "${archive}" "https://downloads.apache.org/kafka/${SNAPCRAFT_PROJECT_VERSION}/${archive}"
       mkdir -p $CRAFT_PART_INSTALL/opt/kafka
       tar -xzvf "${archive}" -C "${CRAFT_PART_INSTALL}/opt/kafka" --strip-components=1
+  kafka-exporter:
+    plugin: nil
+    override-build: |
+      exporter="kafka_exporter-1.6.0.linux-amd64.tar.gz"
+      curl -sLo "${exporter}" "https://github.com/danielqsj/kafka_exporter/releases/download/v1.6.0/${exporter}"
+      tar -xvf "${exporter}" -C "${CRAFT_PART_INSTALL}/" --strip-components=1
```

Comment:
Pulling random binaries directly from GitHub feels unsafe.

I've pinned a version to help alleviate it somewhat, but do please share thoughts.

Reply:
It is fine. In the long term we should rather build this component from source and control/support their built. For the time being, maybe we can also use hashes to be even safer, like [here](https://github.com/canonical/spark-rock/blob/main/rockcraft.yaml#L23)